lib/components/package.json

```diff
@@ -53,8 +54,12 @@
   },
   "devDependencies": {
     "@popperjs/core": "^2.6.0",
+    "@testing-library/jest-dom": "^5.16.4",
     "@types/color-convert": "^2.0.0",
+    "@types/lodash": "^4.14.182",
     "@types/overlayscrollbars": "^1.12.0",
+    "@types/qs": "^6.9.7",
+    "@types/util-deprecate": "^1.0.0",
```

Comment:
@shilman not sure if I should add these, but without them `tsc` was complaining.